docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
```

Comment:
@jannickfahlbusch then you create a dependency on the host system. and what about docker on osx?